common/main/java/com/couchbase/lite/ResultSet.java

```diff
@@ -114,7 +114,7 @@ else if (!c4enum.next()) {
         }
 
         // Log outside the the synchronized block
-        Log.w(DOMAIN, msg, err);
+        Log.i(DOMAIN, msg, err);
```

Comment:
Shouldn't be warn or error level?

Reply:
I think it shouldn't be there at all.  I was going to ask you if I could remove it all together.  It is entirely reasonable, it seems to me, to run this thing until it returns `null`.